src/Logging/src/DynamicLogger/DynamicConsoleLoggerProvider.cs

```diff
@@ -194,23 +171,23 @@ protected virtual void Dispose(bool disposing)
                     // Cleanup
                     _delegate?.Dispose();
                     _delegate = null;
-                    _settings = null;
                     _loggers = null;
                 }
 
                 disposed = true;
             }
         }
 
-        ~DynamicLoggerProvider()
+        ~DynamicConsoleLoggerProvider()
```

Comment:
It's a little strange to see a finalizer here. You don't have any unmanaged resources